src/constants/blocklist.ts

```diff
@@ -0,0 +1,4 @@
+// List of delisted projects
+export const BLOCKED_PROJECT_IDS = [
+  '564', //copycat of 563
```

Comment:
we'll want these to be project `id`s, not `projectId`s (otherwise this would block a v1 project 564). we also want it to be specific to mainnet

Reply:
or, we could copy the pattern used for archived projects (storing `V2ArchivedProjectIdsByNetwork` that are all pv == 2), but we need to add the `pv == 2` statement in the filter

Reply:
<img width="341" alt="image" src="https://github.com/jbx-protocol/juice-interface/assets/79433522/a7e2cf1d-69c1-4075-8ca7-3bab64e5fd90">
<img width="641" alt="image" src="https://github.com/jbx-protocol/juice-interface/assets/79433522/b398de75-c839-4cb0-a8bf-e303f515e734">